### PR TITLE
Refactor/bugs and details

### PR DESCRIPTION
**Pull request by philalewis**
What (if any) features are you implementing?
- Mostly a refactor

What (if anything) did you refactor?
- User is navigated back to home screen if they go to the scorecard before a round has started
- Remove players from the dropdown options once they have been selected in the setup form
- Show distances for each hole and entire course
- Show par on every hole
- Reset scores to par when starting a new hole
- Able to navigate around site and back to scorecard
- Add button to navigate home on the round overview page
- Added search bar to Courses page and make it work
- Make logo the home button

Were there any issues that arose?
- yes

Is there anything that you need from your teammate?
- Please review and merge

Any other comments, questions, or concerns?
- no

### CODE REVIEW

File: src/Components/CourseHeader.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../Styles/CourseHeader.scss'
+
+const CourseHeader = (props) => {
+  return (
```
Comment (oacortez) [inline src/Components/CourseHeader.js:5]: Great function to show our users their names, city, and state

File: src/Components/Home.js [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
 import Searchbar from './Searchbar'
 import '../Styles/Home.scss'
 
 const Home = () => {
   const { leagueMembers } = useContext(LeagueMembers)
 
-  const members = leagueMembers.map(member => {
-    return <p key={member.id}>{member.name}</p>
-  })
+  const members = () => {
```
Comment (oacortez) [inline src/Components/Home.js:9]: great use of sort!

File: src/Components/Scorecard.js [hunk 3]
```diff
@@ -2,30 +2,49 @@ import React, { useContext, useState, useEffect } from 'react'
...
+  const { holeNumber, setHoleNumber } = useContext(HoleNumber)
+  const { errorMessage, setErrorMessage } = useContext(Errors)
+  const [ inProgress, setInProgress ] = useState(true)
+  const { scores, setScores } = useContext(RoundScores)
+  const { courseInfo, setCourseInfo } = useContext(CourseInfo)
   const navigate = useNavigate()
 
-
   useEffect(() => {
+    if (!scorecard.length) {
```
Comment (oacortez) [inline src/Components/Scorecard.js:24]: great use of a conditional!

File: src/Components/ScorecardForm.js [hunk 4]
```diff
@@ -1,32 +1,42 @@
...
+      return {
+        value: member,
+        label: member.name
+      }
+    }))
+  }, [])
 
   const handleChange = event => {
+    const newOptions = options.filter(option => {
+      console.log('option>>>', option, 'event.value>>>>', event.value)
```
Comment (oacortez) [inline src/Components/ScorecardForm.js:30]: will need to remove line 30

**Review by oacortez — APPROVED** [review]
Thank you for making a lot of changes!

File: src/Components/Scorecard.js [hunk 5]
```diff
@@ -106,9 +117,34 @@ const Scorecard = () => {
     .catch(error => setErrorMessage(error))
   }
 
+  // const displayItems = scorecard.length > 0 ?
```
Comment (GraceGardner) [inline src/Components/Scorecard.js:120]: Not sure if we still need this

File: src/Components/ScorecardForm.js (hunk 4 above)
Comment (GraceGardner) [reply src/Components/ScorecardForm.js:30]: I agree with this one :)

**Pull request merged**